src/views/tips/tips.scss

```diff
@@ -31,25 +31,23 @@ $base-bg: $ui-white;
     max-width: calc(100% - 2rem);
 }
 
-$darken-button: rgba(0, 0, 0, .1);
-
 .tips-button {
     margin-right: .75rem;
     background-color: $ui-blue;
     color: $ui-white;
     font-size: 1rem;
-    
+
     &.getting-started-button {
         margin-right: 0;
-        background-color: $darken-button;
+        background-color: $ui-aqua;
```

Comment:
The button here looks like an actual button now (which it didn't really before), but it seems like it might be a little too high contrast?
<img width="417" alt="screenshot 2018-07-31 15 01 03" src="https://user-images.githubusercontent.com/3019167/43480919-94890a54-94d2-11e8-8d8f-58d3cf18df81.png">